Sofascore2025/Sofascore2025/View/MenuItemView.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  MenuItemView.swift
+//  Sofascore2025
+//
+//  Created by Ivona Jelovic on 21.03.2025..
+//
+
+import UIKit
+import SnapKit
+
+class MenuItemView: UIView {
+
+    private let imageView = UIImageView()
+    private let titleLabel = UILabel()
+
+    init(text: String, image: String) {
```

Comment:
radije bi da ove stvari ne predajemo u init-u, napravi public metode npr ovako nesto
```suggestion
    @discardableResult
    func title(_ text: String) -> Self {
        titleLabel.text(text)
        return self
    }
```

Reply:
i tako nesto mozes odvojit u ekstenziju npr ovako
```
// MARK: - Public methods
extension MenuItemView {
```